ale/rotation.py

```diff
@@ -217,12 +217,11 @@ def inverse(self):
         Get the inverse rotation, that is the rotation from the destination
         reference frame to the source reference frame.
         """
-        new_rots = self._rots.inv()
         if self.av is not None:
-            new_av = -new_rots.apply(self.av)
+            new_av = -self._rots.apply(self.av)
```

Comment:
This is the same as the fix in the AV PR. I had defined the angular velocity as the speed at which frame 1 rotates with respect to frame 2. This and the transposes in the velocity apply steps fixes that.